cnd/src/btsieve/ethereum.rs

```diff
@@ -6,14 +6,14 @@ use crate::{
     btsieve::{
         find_relevant_blocks, BlockByHash, BlockHash, LatestBlock, Predates, PreviousBlockHash,
     },
-    ethereum::{Address, Bytes, Input, Log, Transaction, TransactionReceipt, H256, U256},
+    ethereum::{Address, Bytes, Input, Log, Transaction, TransactionReceipt, U256},
 };
 use anyhow;
 use async_trait::async_trait;
 use chrono::NaiveDateTime;
 use genawaiter::{sync::Gen, GeneratorState};
 
-type Hash = H256;
+type Hash = crate::ethereum::Hash;
 type Block = crate::ethereum::Block;
```

Comment:
True, I left it because we basically still do the same for the Block. Originally it was to properly map the types so it is easier to read. Can clean this up for both and just import.